packages/back-end/src/controllers/datasources.ts

```diff
@@ -39,7 +39,7 @@ import {
 import uniqid from "uniqid";
 
 export async function postSampleData(req: AuthRequest, res: Response) {
-  req.checkPermissions("createMetrics");
+  req.checkPermissions("createMetrics", "");
   req.checkPermissions("createAnalyses", "");
```

Comment:
checkPermissions is an overloaded interface. It's kinda confusing...
- with 1 parameter, it's checking global permissions (which is incorrect for "createMetrics" which is now a project scoped permission)
- with 2 parameters, it's checking project scoped permissions. The 2nd parameter is either a specific project string OR "" (which indicates we must have this permission scoped globally despite it being project-specific ... like an admin user for instance)
- with 3 parameters, it's checking environment scoped permissions (and also allows for projects).

Reply:
I see what you mean. This seems like a TypeScript issue since `checkPermissions` is supposed to accept `project`  of type `string | undefined` for project and environment scoped permissions ... Nevertheless, definitely okay with tabling it for later.